src/styles/utils/card-block.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  transition: all 0.3s ease;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__img {
+    width: 100%;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+  &__prod {
+    margin-bottom: 16px;
+  }
+  &__review {
+    margin-bottom: 24px;
+  }
+  &__price {
+    margin-bottom: 16px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+}
+
+.product {
+  h3,
+  p {
+    margin: 0;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 150%;
+    color: v.$col-title-card;
+    margin-bottom: 4px;
+    transition: all 0.3s ease;
+  }
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
+    color: v.$col-grey;
+  }
+}
+
+.card:hover .product__title {
+  color: v.$col-title-card-hover;
```

Comment:
Ensure that the variable `v.$col-title-card-hover` is defined in your variables file. If it's not defined, this will cause an error during compilation.